src/components/Header/Header.js

```diff
@@ -1,11 +1,12 @@
-import React, { useMemo } from 'react'
+import React, { useCallback, useMemo } from 'react'
 import { useHistory } from 'react-router'
-import { GU, Link, useTheme, useViewport } from '@1hive/1hive-ui'
+import { GU, Link, useTheme, useViewport, ButtonBase } from '@1hive/1hive-ui'
```

Comment:
Let's alphabetize this import
```suggestion
import { ButtonBase, GU, Link, useTheme, useViewport } from '@1hive/1hive-ui'
```